commands.py

```diff
@@ -215,6 +215,9 @@ def handle_kick(state: mantatail.ServerState, user: mantatail.UserConnection, ar
 
     channel.kick_user(user, target_usr, message)
 
+    if target_usr in channel.operators:
+        channel.operators.discard(target_usr)
```

Comment:
`discard` does nothing if the user is not in the set. This is how `discard` and `remove` differ.
```suggestion
    channel.operators.discard(target_usr)
```